lean/commands/live.py

```diff
@@ -776,7 +781,7 @@ def live(project: Path,
                                                         gdax_use_sandbox)
         elif brokerage == BinanceBrokerage.get_name():
             ensure_options(["binance_api_key", "binance_api_secret", "binance_use_testnet"])
-            brokerage_configurer = BinanceBrokerage(binance_api_key, binance_api_secret, binance_use_testnet)
+            brokerage_configurer = BinanceBrokerage(binance_organization, binance_api_key, binance_api_secret, binance_use_testnet)
```

Comment:
Should be doing something like `_get_organization_id(zerodha_organization, "Zerodha")` instead? Same bellow